README.md

```diff
@@ -1 +1,38 @@
-# code-review
\ No newline at end of file
+
+# hello-project
+
+> My first project
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Usage
+- Open the .html
+
+## Authors
+
+👤 **Author1**
```

Comment:
it would be great if you write your name instead of **Author1** here

Reply:
> it would be great if you write your name instead of **Author1** here

- [ ] Great job 😃 at using the inline comments, but kindly tell the reason for the change to the student because this may be obvious to you and not clear to the student for example tell the student 
``` it will be better 😊 if you provide your information here like your name and the accounts to let the visitors know who are you? 😃```💯